servers/rendering/renderer_rd/shaders/canvas.glsl

```diff
@@ -117,13 +135,10 @@ void main() {
 
 	mat4 model_matrix = mat4(vec4(draw_data.world_x, 0.0, 0.0), vec4(draw_data.world_y, 0.0, 0.0), vec4(0.0, 0.0, 1.0, 0.0), vec4(draw_data.world_ofs, 0.0, 1.0));
 
-#define FLAGS_INSTANCING_MASK 0x7F
-#define FLAGS_INSTANCING_HAS_COLORS (1 << 7)
-#define FLAGS_INSTANCING_HAS_CUSTOM_DATA (1 << 8)
+#ifdef USE_ATTRIBUTES
```

Comment:
Moved to here, as `USE_ATTRIBUTES` the FLAGS_INSTANCING_MASK is only used when using vertex attributes